AdyenCard/Components/GiftCardComponent/GiftCardComponent.swift

```diff
@@ -183,24 +205,36 @@ public final class GiftCardComponent: PartialPaymentComponent,
 
         startLoading()
 
-        let data = createPaymentData()
+        fetchCardPublicKey(discardError: false) { [weak self] cardPublicKey in
+            guard let self = self else { return }
+            let paymentDataResult = self.createPaymentData(order: self.order, cardPublicKey: cardPublicKey)
```

Comment:
maybe this line is not needed, can do `self.createPaymentData(order: self.order, cardPublicKey: cardPublicKey).mapError...`

Reply:
👍